AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -15,6 +15,7 @@
 ##head[prefix~="medium-com:"] ~ body .js-openInAppButton
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
+sp.live.nicovideo.jp##a[class*="app-download-banner"]
```

Comment:
Should be removed as per https://github.com/AdguardTeam/AdguardFilters/pull/107473 but alternatively you can close the old PR and merge the rules to this PR.

Reply:
ok